apollo-router/src/plugins/telemetry/formatters/mod.rs

```diff
@@ -109,7 +109,7 @@ pub(crate) fn filter_metric_events(event: &tracing::Event<'_>) -> bool {
     })
 }
 
-pub(crate) fn to_map(resource: Resource) -> HashMap<String, serde_json::Value> {
+pub(crate) fn to_vec(resource: Resource) -> LinkedList<(String, serde_json::Value)> {
```

Comment:
I know you mean `to_list`...

Reply:
Oops, it has been merged, here is the fix https://github.com/apollographql/router/pull/4257